onnx/version_converter.py

```diff
@@ -26,7 +26,7 @@
     RuntimeError when some necessary conversion is not supported
 
 Supported adapters:
-    --
+    --Add from Opset 7 to Opset 6
```

Comment:
Add all supported adapters here?

Also an unsupported section?

Reply:
Should the unsupported by explicit (i.e. should I go through and identify all of these)?

Reply:
unsupported part is more for documentation purpose. So later we know which adapter we need to implement